pkgs/applications/networking/sniffers/wireshark/default.nix

```diff
@@ -108,11 +109,11 @@ stdenv.mkDerivation {
     c-ares
     glib
     zlib
-  ] ++ lib.optionals withQt (with qt5; [ qtbase qtmultimedia qtsvg qttools ])
-  ++ lib.optionals (withQt && stdenv.isLinux) [ qt5.qtwayland ]
+  ] ++ lib.optionals withQt (with qt6; [ qtbase qtmultimedia qtsvg qttools qt5compat ])
+  ++ lib.optionals (withQt && stdenv.isLinux) [ qt6.qtwayland ]
   ++ lib.optionals stdenv.isLinux [ libcap libnl sbc ]
   ++ lib.optionals stdenv.isDarwin [ SystemConfiguration ApplicationServices gmp ]
-  ++ lib.optionals (withQt && stdenv.isDarwin) (with qt5; [ qtmacextras ]);
+  ++ lib.optionals (withQt && stdenv.isDarwin) (with qt6; [ qtmacextras ]);
```

Comment:
Looks like `qtmacextras` does not exist as a module in Qt6. Does this break WireShark on macOS using Qt6 or is this no longer required with the new version?

Reply:
> Looks like `qtmacextras` does not exist as a module in Qt6. Does this break WireShark on macOS using Qt6 or is this no longer required with the new version?

Qtmacextra [no longer exists](https://www.qt.io/blog/qt-extras-modules-in-qt-6) in qt6, this line should be deleted.

Reply:
Removed that line. Sadly I have no macOS to test the Darwin build on.